jekyll/_cci2/env-vars.md

```diff
@@ -16,19 +16,39 @@ This document describes using environment variables in CircleCI in the following
 
 ## Overview
 
-Environment variables are used according to a specific precedence order, as follows:
-
-1. Environment variables declared inside a shell command in a `run` step, for example `FOO=bar make install`.
-2. Environment variables declared with the `environment` key for a `run` step.
-3. Environment variables set with the `environment` key for a job.
-4. Environment variables set with the `environment` key for a container.
-5. Context environment variables (assuming the user has access to the Context). See the [Contexts]( {{ site.baseurl }}/2.0/contexts/) documentation for instructions.
-6. Project-level environment variables set on the Project Settings page.
-7. Special CircleCI environment variables defined in the [CircleCI Environment Variable Descriptions]({{ site.baseurl }}/2.0/env-vars/#circleci-built-in-environment-variables) section of this document.
-
-Environment variables declared inside a shell command `run step`, for example `FOO=bar make install`, will override environment variables declared with the `environment` and `contexts` keys. Environment variables added on the Contexts page will take precedence over variables added on the Project Settings page. Finally, special CircleCI environment variables are loaded.
-
-## Adding Project-Level Environment Variables
+Environment variables can be set at several scope levels.
+From general to specific, these levels are:
+
+1. Environment variables [built into CircleCI](#circleci-built-in-environment-variables).
+Automatically available.
+2. [Project-level](#setting-project-level-environment-variables) environment variables.
+Set within the CircleCI application.
+3. [Context-level]({{ site.baseurl }}/2.0/contexts/) environment variables.
+Set within the CircleCI application.
+4. [Job-level](#setting-job-level-environment-variables) environment variables.
+Set in your `.circleci/config.yml` file.
+5. [Container-level](#setting-container-level-environment-variables) environment variables.
+Set in your `.circleci/config.yml` file.
+6. [Step-level](#setting-step-level-environment-variables) environment variables.
+Set in your `.circleci/config.yml` file.
+7. Environment variables [set within shell commands](#setting-command-level-environment-variables).
+Set in your `.circleci/config.yml` file.
+
+CircleCI also loads environment variables in this order.
+If an environment variable is set at multiple scope levels,
+CircleCI uses the variable set at the **lowest level**.
+
+**Warning**:
+Environment variables set within configuration (levels 4-7)
+are **fully visible** in build output.
```

Comment:
I would change `in build output` to `in build output and on the build page`.

Reply:
Is it more correct to say `in built output on the build page`? Where are all the possible places for build output?

Reply:
You can grab build output from the API too.